packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -109,62 +115,51 @@ export default function ListViewSidebar( { listViewToggleElement } ) {
 		handleToggleListViewShortcut
 	);
 
-	/**
-	 * Render tab content for a given tab name.
-	 *
-	 * @param {string} tabName The name of the tab to render.
-	 */
-	function renderTabContent( tabName ) {
-		if ( tabName === 'list-view' ) {
-			return (
-				<div className="edit-post-editor__list-view-panel-content">
-					<ListView dropZoneElement={ dropZoneElement } />
-				</div>
-			);
-		}
-		return <ListViewOutline />;
-	}
-
 	return (
 		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
 		<div
 			className="edit-post-editor__document-overview-panel"
 			onKeyDown={ closeOnEscape }
 			ref={ sidebarRef }
 		>
-			<Button
-				className="edit-post-editor__document-overview-panel__close-button"
-				icon={ closeSmall }
-				label={ __( 'Close' ) }
-				onClick={ closeListView }
-			/>
-			<TabPanel
+			<div
 				className="edit-post-editor__document-overview-panel__tab-panel"
 				ref={ tabPanelRef }
-				onSelect={ ( tabName ) => setTab( tabName ) }
-				selectOnMove={ false }
-				tabs={ [
-					{
-						name: 'list-view',
-						title: _x( 'List View', 'Post overview' ),
-						className: 'edit-post-sidebar__panel-tab',
-					},
-					{
-						name: 'outline',
-						title: _x( 'Outline', 'Post overview' ),
-						className: 'edit-post-sidebar__panel-tab',
-					},
-				] }
 			>
-				{ ( currentTab ) => (
+				<Tabs
+					selectOnMove={ false }
+					onSelect={ ( tabName ) => setTab( tabName ) }
+				>
+					<Tabs.TabList>
+						<Tabs.Tab tabId="list-view">
+							{ _x( 'List View', 'Post overview' ) }
+						</Tabs.Tab>
+						<Tabs.Tab tabId="outline">
+							{ _x( 'Outline', 'Post overview' ) }
+						</Tabs.Tab>
+					</Tabs.TabList>
+					<Button
+						className="edit-post-editor__document-overview-panel__close-button"
+						icon={ closeSmall }
+						label={ __( 'Close' ) }
+						onClick={ closeListView }
+					/>
+
 					<div
 						className="edit-post-editor__list-view-container"
 						ref={ listViewContainerRef }
 					>
```

Comment:
I wonder if we could get rid of this `div` and somehow pass the relevant classname and ref to the tab panels (or find an alternative solution)?

Reply:
I think we can, if we just pass the classname to the `tabpanel`s themselves they styles shouldn't be negatively impacted. The ref looks, unless I'm mistaken, like it's only relevant to the List View, so we can pass the ref there directly.